static/src/javascripts/projects/common/modules/experiments/tests/commercial-uk-row-mobile-sticky.js

```diff
@@ -0,0 +1,48 @@
+// @flow strict
+import once from 'lodash/once';
+import config from 'lib/config';
+import { isBreakpoint } from 'lib/detect';
+import { getSync as geolocationGetSync } from 'lib/geolocation';
+
+const currentGeoLocation = once((): string => geolocationGetSync());
+const isInUkRegion = (): boolean => currentGeoLocation() === 'GB';
+const isInUsRegion = (): boolean =>
+    ['US', 'CA'].includes(currentGeoLocation());
+const isInAuRegion = (): boolean =>
+    ['AU', 'NZ'].includes(currentGeoLocation());
+const isInRowRegion = (): boolean =>
+    !isInUkRegion() && !isInUsRegion() && !isInAuRegion();
+
+export const commercialUkRowMobileSticky: ABTest = {
+    id: 'CommercialUkRowMobileSticky',
+    start: '2020-04-15',
+    expiry: '2020-04-22',
+    author: 'George Haberis',
+    description:
+        'Test mobile sticky ad in UK and ROW regions',
+    audience: 0.01,
+    audienceOffset: 0.8, // To avoid clashes with other tests and the results on this AB test are already very sensitive to outside factors
+    successMeasure: 'No significant impact to performance as well as higher ad yield',
+    audienceCriteria: 'n/a',
+    dataLinkNames: 'n/a',
+    idealOutcome:
+        'No significant impact to performance as well as higher ad yield',
+    showForSensitive: true,
+    canRun: () =>
```

Comment:
Having `config.get('page.contentType') === 'Article' && !config.get('page.isHosted')` as part of the test's canRun means users will be removed from the test if they access a page that's not an article or if its hosted content, will it not? This would have adverse affects when looking at the data because we wouldn't be able to tell if the user saw the mobilesticky through their whole journey (when applicable) or just part of it, making it very difficult to extract any conclusions.
My suggestions it to have this check as part of the `shouldIncludeMobileSticky()` function, which it already has.

Reply:
Updated as suggested 👍